teams/cloudbees-developers.yml

```
@@ -17,3 +15,5 @@ developers:
 - "teilo"
 - "twasyl"
 - "vlatombe"
+- "fcojfernandez"
+- "PereBueno"
```

```suggestion
- "PereBueno"

```
NIT: Adding a newline